apps/ca.c

```diff
@@ -1817,8 +1817,6 @@ static int do_body(X509 **xret, EVP_PKEY *pkey, X509 *x509,
 
             if (push != NULL) {
                 if (!X509_NAME_add_entry(subject, push, -1, 0)) {
-                    if (push != NULL)
-                        X509_NAME_ENTRY_free(push);
```

Comment:
You still need to free it.  It's not freed down by `err:`

Reply:
As far as I understand, it's freed among other stuff in X509_REQ_free(rreq) at apps/ca.c:1559. It's the same behaviour as on master.

Reply:
Yes, this is the same as per #4698